integration-tests/README.md

```diff
@@ -0,0 +1,107 @@
+# Integration tests for Oracle WebLogic Server Kubernetes Operator
+This documentation describes the steps to run integration tests for the Oracle WebLogic Server Kubernetes Operator locally on a Oracle Linux environment. The tests are written in the JUnit5 framework and driven by maven profile.
+
+# Directory Configuration and Structure
+Directory structure of integration test source is distributed as follows 
+```
+weblogic-kubernetes-operator/integration-tests - location of module pom.xml  
+weblogic-kubernetes-operator/integration-tests/src/test/java/oracle/weblogic/kubernetes - integration test(JUnit5) classes and utility classes  
+weblogic-kubernetes-operator/integration-tests/src/test/resources - properties, YAML files and other bash scripts
+```
+
+# How does WebLogic Kubernetes Operator integration test work
+ - Build WebLogic Kubernetes Operator image from the downloaded git branch.
+ - Install the latest version of WebLogic Deploy Tooling (WDT) and WebLogic Image Tool (WIT)
+ - Install supported istio service mesh.
+ - Pull the specified version of WebLogic image from container-registry.oracle.com (OCR)
+ - Install WebLogic Kubernetes Operator.
+ - Build a WebLogic domain resource.
+ - Make sure WebLogic domain is running by verifying the corresponding server pod and kubernetes service status. 
```

Comment:
sure WebLogic domain -> sure that the WebLogic domain
kubernetes  -> Kubernetes

Reply:
done